packages/docs-site/package.json

```diff
@@ -57,14 +57,15 @@
     "react-testing-library": "^5.6.1",
     "rehype-react": "^3.1.0",
     "rimraf": "^2.6.3",
+    "sass": "^1.20.1",
     "source-code-pro": "^2.30.1",
     "styled-components": "^4.1.3",
     "stylelint": "^9.10.1",
     "stylelint-junit-formatter": "^0.2.1",
     "uuid": "^3.3.2"
   },
   "resolutions": {
-    "firebase-tools/**/lodash" : "4.17.11"
+    "firebase-tools/**/lodash": "4.17.11"
```

Comment:
Did adding lodash explicitly actually fix the resolution issue?

Reply:
This has happened automatically when I did a `yarn add lodash`.

I'm using lodash in that sidebar component - I use the sortBy helper.